current-events/activity-current-events.instructor.md

```diff
@@ -38,3 +57,22 @@ It's intentional that the format of this is small conversations, not a large gro
 ## Meta
 
 The list of triggers comes from [a trigger warning guide](https://trigger-warning-guide.tumblr.com/triggers). I pruned some topics from my own POV.
+
+### !callout-info
+
+## Facilitation Notes to Consider: Ada's Company Partners and "Don't Yuck My Yum"
+
+The trickiest balance of this entire thing is to center the activity around discussion of current events outside our control. Students are doing the right thing by acknowledging the current events exist.
+
+<br/>
+
+I anticipate that many current events are about Ada company partners, and are critical of them. It's important to _not_ center around "is this a bad company, and what if I or someone I know interns at this bad company?" because those things are outside individual control. Also, discussion should not lead to the judgment of others. Bringing an article decrying Google (example) should not mean that they hate Google and everyone who works at Google, and that they never want to work at Google.
+
+<br/>
+
+If appropriate, it may be helpful to bring up:
+
+- Working at a tech giant happens! You survive and thrive!! And it's your individual journey to figure out how you fit into that, but being employed is always a good thing when surviving under capitalism.
+- "Don't yuck my yum"-- bring up good news, but don't knock me for liking something
```

Comment:
"Don't yuck my yum" is a phrase I was unfamiliar with (not sure about the rest of the instructional staff). I would have assumed in the context of the rest of the material here that it would be more like it's ok for someone to bring up _bad_ news about a company, product, etc, but you don't need to make me feel bad about liking some aspect of that same company, product, etc.

Reply:
OOooh! Yeah, I learned the phrase "Don't yuck my yum" from another Ada staff member, but that was a while ago!

I'm removing the phrase because you're totally right-- the meaning isn't quite applicable to this :D